blockchain/chain_sync/src/validation.rs

```diff
@@ -118,15 +118,16 @@ impl<'a> TipsetValidator<'a> {
         let bls_cids = bls_msgs
             .iter()
             .map(Cbor::cid)
-            .collect::<Result<Vec<cid_orig::Cid>, fvm_shared::encoding::Error>>()?;
+            .collect::<Result<Vec<Cid>, fvm_shared::encoding::Error>>()?;
         let secp_cids = secp_msgs
             .iter()
             .map(Cbor::cid)
-            .collect::<Result<Vec<cid_orig::Cid>, fvm_shared::encoding::Error>>()?;
+            .collect::<Result<Vec<Cid>, fvm_shared::encoding::Error>>()?;
 
         // Generate Amt and batch set message values
         let bls_message_root = Amt::new_from_iter(blockstore, bls_cids)?;
         let secp_message_root = Amt::new_from_iter(blockstore, secp_cids)?;
+        dbg!(bls_message_root, secp_message_root);
```

Comment:
Is it a leftover from debugging or it is supposed to be here?

Reply:
A leftover. Should be gone now.